front/pages/w/[wId]/assistant/labs/transcripts/index.tsx

```diff
@@ -286,33 +279,30 @@ export default function LabsTranscriptsIndex({
   };
 
   const handleConnectGoogleTranscriptsSource = async () => {
-    try {
-      if (transcriptsConfigurationState.provider !== "google_drive") {
-        return;
-      }
-      const nango = new Nango({ publicKey: nangoPublicKey });
-      const nangoConnectionId = buildLabsConnectionId(
-        `labs-transcripts-workspace-${owner.id}-user-${user.id}`,
-        transcriptsConfigurationState.provider
-      );
-      const {
-        connectionId: newConnectionId,
-      }: { providerConfigKey: string; connectionId: string } = await nango.auth(
-        nangoDriveConnectorId,
-        nangoConnectionId
-      );
+    if (transcriptsConfigurationState.provider !== "google_drive") {
```

Comment:
IIRC, the `try/catch` was used to catch errors coming from `saveOauthConnection` that does a `fetch` call, can we either keep it or move it to the function?

Reply:
Added a try catch